src/test_l1_dsAudio.c

```diff
@@ -4486,41 +5187,52 @@ void test_l1_dsAudio_negative_dsGetAudioLevel(void) {
  * |04|Call dsAudioPortTerm() - Terminate audio ports | | dsERR_NONE |  Termination must be successful |
  * 
  */
-void test_l1_dsAudio_positive_dsSetAudioLevel(void) {
+void test_l1_dsAudio_positive_dsSetAudioLevel_sink(void) {
```

Comment:
where is test case for source ?
dsSetAudioLevel()
check this 
https://github.com/rdkcentral/rdk-halif-device_settings/blob/main/include/dsAudio.h#L1057